icevision/visualize/draw_data.py

```diff
@@ -22,49 +29,115 @@ def draw_sample(
     denormalize_fn: Optional[callable] = None,
     display_label: bool = True,
     display_bbox: bool = True,
+    display_score: bool = True,
     display_mask: bool = True,
     display_keypoints: bool = True,
-):
+    font_path: Optional[os.PathLike] = DEFAULT_FONT_PATH,
+    font_size: Union[int, float] = 12,
+    label_color: Union[np.array, list, tuple, str] = "#C4C4C4",  # Mild Gray
+    mask_blend: float = 0.5,
+    mask_border_thickness: int = 7,
+    color_map: Optional[dict] = None,  # label -> color mapping
+    prettify: bool = True,
+    prettify_func: Callable = str.capitalize,
+    return_as_pil_img=False,
+    # Args for plotting specific labels
+    exclude_labels: List[str] = [],
+    include_only: List[str] = None,
+) -> Union[np.ndarray, PIL.Image.Image]:
+    """
+    Selected kwargs:
+
+    * label_color: A <collection> of RGB values or a hex code string that defines
+                   the color of all the plotted labels
+    * mask_blend: Degree of transparency of the mask. 1 = opaque, 0 = transparent
+    * mask_border_thickness: Degree of thickness of the mask. Must be an odd number
+    * color_map: An optional dictionary that maps the label => color-value
+    * prettify: Format labels based on `prettify_func`
+    * prettify_func: A string -> string processing function
+    * return_as_pil_image: If true, returns the sample as a PIL image, else np.array
+    * exclude_labels: (Optional) List of labels that you'd like to exclude from being plotted
+    * include_only: (Optional) List of labels that must be exclusively plotted. Takes
+                    precedence over `exclude_labels` (?)
+    """
     img = sample["img"].copy()
     if denormalize_fn is not None:
         img = denormalize_fn(img)
 
-    for label, bbox, mask, keypoints in itertools.zip_longest(
+    for label, bbox, score, mask, keypoints in itertools.zip_longest(
         sample.get("labels", []),
         sample.get("bboxes", []),
+        sample.get("scores", []),
         sample.get("masks", []),
         sample.get("keypoints", []),
     ):
+        # random color by default
         color = (np.random.random(3) * 0.6 + 0.4) * 255
-        color = tuple(color.astype(int).tolist())
+
+        # logic for plotting specific labels only
+        # `include_only` > `exclude_labels`
+        if not label == []:
+            label_str = class_map.get_id(label)
```

Comment:
@lgvaz A bunch of the test failures stem from my assumption that a `class_map` is always passed. If this isn't true, I could change this to something like:
```python
label_str = class_map.get_id(label) if class_map else ""    
```